include/cocaine/detail/service/node/cached.hpp

```diff
@@ -64,19 +64,19 @@ cached<T>::cached(context_t& context, const std::string& collection, const std::
 
     try {
         cache = api::storage(context, "cache");
-    } catch(const api::repository_error_t& e) {
+    } catch(const std::system_error& e) {
```

Comment:
Still don't understand :( Don't try to download if there are no "cache" storage?